### ISSUE TITLE
A couple of linux issues (lv2 not working and config location)

### ISSUE THREAD

**Issue by therealfumbles** [dsp56300/gearmulator#199]
Hi

Thanks for your hard work on this project. I just wanted to raise a couple of issues with the plugins on linux.

The linux lv2 is distributed as a bare .so file. LV2 plugins are bundles containing the .so and three .ttl files. As is, it is not recognized by either of the hosts I use.

Also, the config folder is placed in the home folder. It is better practice to place it in either ~/.config or ~/.local/share.

**Comment by foresense** [comment]
Regarding the config location, I made a patch for that over here:
https://github.com/foresense/nur-packages/blob/master/pkgs/gearmulator/better_folders.patch

**Comment by dsp56300** [comment]
Yes its obsolete, folders have been changed and LV2 plugins now include a .ttl
